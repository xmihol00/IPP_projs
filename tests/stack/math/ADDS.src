.IPPcode21

DEFVAR GF@num1
DEFVAR GF@num2

PUSHS int@82
PUSHS int@14
ADDS

POPS GF@num1
WRITE GF@num1
WRITE string@\010

PUSHS float@0x1.54p+5 
PUSHS float@0x1.ccp+4
ADDS

POPS GF@num2
WRITE GF@num2
WRITE string@\010

PUSHS GF@num1
PUSHS GF@num2
FLOAT2INTS
ADDS
POPS GF@num2
WRITE GF@num2
WRITE string@\010

PUSHS GF@num2
INT2FLOATS
PUSHS float@-0x1.53p+6
ADDS
POPS GF@num2
WRITE GF@num2
WRITE string@\010