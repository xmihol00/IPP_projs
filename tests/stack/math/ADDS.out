96
0x1.1d00000000000p+6
167
0x1.4900000000000p+6
